src/components/Database/Database.tsx

```diff
@@ -14,21 +14,23 @@
  * limitations under the License.
  */
 
-import React, { useState, useEffect, useCallback } from 'react';
-import * as firebase from 'firebase/app';
 import 'firebase/database';
```

Comment:
Same question here, is this after sorting?

Reply:
Yep, side effects go to the top, these are in the api.ts, so likely don't need to be here anyway